tests/utils/esx/govc.go

```diff
@@ -42,9 +42,21 @@ const (
 //govc vm.info -vm.ip=10.20.104.62 -json | jq -r .VirtualMachines[].Name
 func RetrieveVMNameFromIP(ip string) string {
 	// log.Printf("Finding VM name from IP Address [%s]\n", ip)
-	cmd := esx.VMInfoByIP + ip + esxcliJSON + esx.VMName
-	out, _ := ssh.InvokeCommandLocally(cmd)
-	return out
+	var dockerHostName string
+	noVMName := "govc: no such VM"
+	maxAttempt := 20
+	waitTime := 2
+	for attempt := 0; attempt < maxAttempt; attempt++ {
+		cmd := esx.VMInfoByIP + ip + esxcliJSON + esx.VMName
```

Comment:
The command will not be changed dynamically, so let's move this statement out of the for loop.

Reply:
@shaominchen I have addressed your comment in the latest commit.